src/webgpu/api/validation/encoding/programmable/pipeline_bind_group_compat.spec.ts

```diff
@@ -777,7 +777,6 @@ g.test('bgl_resource_type_mismatch')
   )
   .params(
     kCompatTestParams
-      .beginSubcases()
```

Comment:
our skips are inside `fn` so you don't have to remove the `beginSubcases()`, right?

Reply:
thanks for catching that. I deleted that by mistake as I had commented it out to run just one subcase. Will fix.

Reply:
fixed